libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/Same.hs

```diff
@@ -404,7 +403,7 @@ sameShelleyTxBody proof (ShelleyTxBody i1 o1 c1 (Withdrawals w1) f1 s1 pu1 d1) (
   , ("Fee", eqVia pcCoin f1 f2)
   , ("TimeToLive", eqVia pcSlotNo s1 s2)
   , ("PPupdate", eqVia (\_ -> ppString "Update") pu1 pu2)
-  , ("AuxDataHash", eqVia (ppStrictMaybe (\(AuxiliaryDataHash h) -> trim (ppSafeHash h))) d1 d2)
+  , ("AuxDataHash", eqVia (ppStrictMaybe (\(TxAuxDataHash h) -> trim (ppSafeHash h))) d1 d2)
```

Comment:
I know it doesn't matter too much, but just in case you want to update the label too, I'm adding the suggestions, but please feel free to ignore them 
```suggestion
  , ("TxAuxDataHash", eqVia (ppStrictMaybe (\(TxAuxDataHash h) -> trim (ppSafeHash h))) d1 d2)
```